presto-main/src/main/java/com/facebook/presto/security/AllowAllSystemAccessControl.java

```diff
@@ -187,15 +187,4 @@ public void checkCanGrantTablePrivilege(Identity identity, Privilege privilege,
     public void checkCanRevokeTablePrivilege(Identity identity, Privilege privilege, CatalogSchemaTableName table, PrestoPrincipal revokee, boolean grantOptionFor)
     {
     }
-
```

Comment:
Doesn't see this change in https://github.com/prestosql/presto/pull/90/commits/4f4b268adce8a214b73344a82c872bd493ef55ac . Maybe they already implement in different commit?

Reply:
Recovered in `Add access control checks for SHOW ROLES`